force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -0,0 +1,39 @@
+import { LightningElement, api } from "lwc";
+import accountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+
+export default class EdaSettingsNavigation extends LightningElement {
+    labelReference = {
+        accountModelSettingsTitle,
+    };
+
+    settingsNavigationViewModel = {
+        navigationSections: [
+            {
+                label: this.labelReference.accountModelSettingsTitle,
+                page: "accountmodelsettings",
+                sectionkey: "accountmodelsettings",
```

Comment:
very minor thing but sectionKey?

Reply:
Not all navigation sections are navigable. Key can be a keyword in some instances, so sectionkey allows us to have a unique identifier on it.

Reply:
Changed to id in the full nav.